### DISCUSSION TITLE
timer.freeze()

### DISCUSSION THREAD

**Discussion by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#174] [Ideas]
Добавить метод freeze() к таймеру.
Метод будет замораживать остоновку таймера. Например:
`let timer = Timers.getContext().Get('T1');
timer.restartLoop(1);
timer.freeze();
timer.Stop() // функция не будет вызвана`

**Comment by forwork10** [comment]
ну тут нет бага, все предложения нужно в дискуссии постить...

**Comment by kkohno** [comment]
лучше в дискуссии - там проще с предложениями... да и с предложением тут не совсем понятно
